shared.d.ts

```diff
@@ -64,4 +64,15 @@ declare module 'libsession_util_nodejs' {
   }
 
   export type BaseWrapperActionsCalls = MakeWrapperActionCalls<BaseConfigWrapper>;
+
+  export type ConstantsType = {
+    /** 100 bytes */
+    CONTACT_MAX_NAME_LENGTH: number;
+    /** 100 bytes - for legacy groups and communities */
+    BASE_GROUP_MAX_NAME_LENGTH: number;
+    /** 100 bytes */
+    GROUP_INFO_MAX_NAME_LENGTH: number;
+  }
```

Comment:
nice to have the jsdoc, but also easy to get out of sync if ever libsession changes it

Reply:
Yeah not an ideal solution if we can find a way to link or reference the C++ directly then I'm all for it.